context-config-store/scripts/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
can we stick a readme here

Reply:
You mean in the build.sh script or in the sub folder? To explain what this store is doing?

Reply:
not for the script but in the package / subfolder

Reply:
Added a REAME to the contexts-config-store module :)